python_transport/wirepas_gateway/transport_service.py

```diff
@@ -159,6 +159,68 @@ def initialize_sink(self, name):
                 sink.cost = self.minimum_sink_cost
 
 
+class SendGatewayStatusThread(Thread):
+
+    def __init__(
+        self,
+        period,
```

Comment:
should we have a default parameter for the period?

Reply:
period should have unit (s/ms)?

Reply:
Yes, I'm going to add it explicitly